classes/Customer.php

```diff
@@ -1346,11 +1346,13 @@ public function getLastEmptyCart($withOrder = true)
     }
 
     /**
-     * Validate controller.
+     * Validate controller and check password
      *
      * @param bool $htmlentities
      *
      * @return array
+     * 
+     * @deprecated The password check has been moved in controllers and this method is not called anywhere.
```

Comment:
```suggestion
     * @deprecated Deprecated since PrestaShop 1.7.0
```

Reply:
since PS 8.1 ? maybe

Reply:
Hi guys @kpodemski @FabienPapet I added the deprecate, so it depends on the version you will accept the PR, so I can't guess it ? Please help.

Reply:
I think as it is in develop, target is 8.1 as we can't deprecate features on released versions.

Reply:
Yes, obviously 8.1, sorry :D it is not used since 1.7, deprecated since 8.1 👍

Reply:
@kpodemski @FabienPapet it's changed, thanks.